test/integration/start_stop_delete_test.go

```diff
@@ -60,14 +59,8 @@ func TestStartStop(t *testing.T) {
 				"--extra-config=kubelet.network-plugin=cni",
 				"--extra-config=kubeadm.pod-network-cidr=192.168.111.111/16",
 			}},
-			{"containerd", constants.DefaultKubernetesVersion, []string{
-				"--container-runtime=containerd",
-				"--docker-opt",
-				"containerd=/var/run/containerd/containerd.sock",
-				"--apiserver-port=8444",
-			}},
-			{"crio", "v1.15.7", []string{
-				"--container-runtime=crio",
+			{"default-k8s-version", constants.DefaultKubernetesVersion, []string{}},
```

Comment:
apiserver-port=8444 is missing, 
that test is testing default kubernets verison with a different apiserver_port

we can change the test name to
"default-k8s-different-port"

Reply:
done